mdata/store_cassandra_test.go

```diff
@@ -0,0 +1,43 @@
+package mdata
+
+import (
+	"fmt"
+	"testing"
+)
+
+type testCase struct {
+	ttl                uint32
+	windowFactor       int
+	nameFormat         string
+	expectedTableName  string
+	expectedWindowSize uint32
+}
+
+const oneDay = 24 * 60 * 60
+const oneMonth = oneDay * 30
+const oneYear = oneMonth * 12
+
+func TestGetTTLTables(t *testing.T) {
+	tcs := []testCase{
+		{oneDay, 20, "metric_%d", "metric_16", 1},
+		{oneMonth, 20, "metric_%d", "metric_512", 26},
+		{oneYear, 20, "metric_%d", "metric_8192", 410},
+		{oneDay, 50, "metric_%d", "metric_16", 1},
+		{oneMonth, 50, "metric_%d", "metric_512", 11},
+		{oneYear, 50, "metric_%d", "metric_8192", 164},
```

Comment:
I should update the table in the comments to add a case for `TTL`s that are `0`-`1` hours